tests/core/gui/test_main_window.py

```diff
@@ -110,6 +110,8 @@ def _workspace_button_ready():
         assert w_w.layout_workspaces.count() == 1
         wk_button = w_w.layout_workspaces.itemAt(0).widget()
         assert not isinstance(wk_button, QtWidgets.QLabel)
+        # Important: this is necessary for the incoming click to reliably work
```

Comment:
Not sure why this is important: this check box enable workspace mounting, however I don't think we test the mountpoint in GUI test
(on top of that I don't remember the rule for a new workspace to be mounted or not...)

Reply:
It is important as left click is disabled if the check box is checked (i.e the workspace is not mounted). This was the reason why some tests were inconsistent, when the click happened to be sent before the the check box was enabled.